### PR TITLE
Fix issues with Linux

### PR DESCRIPTION
**Pull request by chrisbroome**
- Use white icon
- Set context menu for tray icon
- Add "Main" context menu item to toggle the main window

### CODE REVIEW

**Comment by leo** [comment]
@chrisbroome Awesome, thank you! Does this make Now Desktop fully work on Linux? 😊

**Comment by chrisbroome** [comment]
@leo There's still a minor graphical glitch with the top few pixels of the main window. Also, the main window doesn't appear directly underneath the tray icon like it does on MacOS. But other than that, it works fine.

The primary goal was to make the context menu appear when the tray icon is clicked. Currently only a tiny empty menu entry displays, and it's not obvious that clicking it brings up the main window.

**Comment by matheuss** [comment]
@chrisbroome first of all, thank you for your time 🙏 

Some of us at ZEIT don't have immediate access to a Linux graphic interface (while some of us use Linux as their main OS, almost everyone works from on Apple computers).

Because of that, could you share a few screenshots so we could better evaluate this and then, for example, prioritize granting someone access to a Linux Desktop so we can release proper Linux support? 🙌

**Comment by pckilgore** [comment]
Functionality on this patch set is great for Linux (Ubuntu 18.04 / Budgie).  Right now, it assumes you have your panel/taskbar at the bottom of the screen which isn't necessarily true.  Also a bit ignorant of gtk them being dark or not, but that's not an issue at all. 

@matheuss -- screenshots from my oss account below.
screenshot from 2018-06-29 17-03-31
screenshot from 2018-06-29 16-57-29
screenshot from 2018-06-29 16-58-02

A couple things: 
 - I received the  "deployed!" system notification when deploying from the GUI but wasn't fast enough to catch it.
 - The GUI has some lag updating after actions are taking.  Hard for me to tell if this is normal since I'm mostly a CLI user.
 - I'm able to exit the application and change settings without issues from the GUI.

Thanks @chrisbroome ! :thumb: :shipit:

**Comment by chrisbroome** [comment]
Sorry for the delayed response here. I noticed this branch is also out of date. I'll rebase, resolve the conflicts, and repush it. If it's of note, I'm still running Ubuntu 16.04 LTS.

**Comment by rauchg** [comment]
No rush! We're very thankful for your contributions

**Comment by chrisbroome** [comment]
OK, I rebased and pushed my changes. The diffs are definitely much easier to read when "hide whitespace changes is enabled".

File: main/index.js [hunk 1]
```diff
@@ -218,6 +221,11 @@ app.on('ready', async () => {
     }
   }
 
+  // Linux requires setContextMenu to be called in order for the context menu to populate correctly
+  tray.setContextMenu(
+    loggedIn ? await contextMenu(windows) : outerMenu(app, windows)
+  )
```
[the remark this answers is not in the document]
Comment (leo) [reply main/index.js]: Truth!

**Comment by leo** [comment]
Thank you so much!

File: main/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (morajabi) [reply main/index.js]: 😅👌

**Pull request merged**